### DISCUSSION TITLE
Where is the syntax highlighting file for Go?

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/ecode#182] [General]
Adept is based on Go. To make Adept syntax highlighting, I will need to copy from Go. But I can't find Go syntax highlighting file in https://github.com/SpartanJ/eepp/tree/develop/src/eepp/ui/doc/languages. I checked and confirmed that I still get syntax highlighting for Go source file. So, where is the syntax highlighting file for Go?

**Comment by SpartanJ** [comment]
It's [here](https://github.com/SpartanJ/eepp/blob/develop/src/eepp/ui/doc/syntaxdefinitionmanager.cpp#L738). I didn't move it to its own file yet.
